apps/api/src/app/inbox/dtos/get-notifications-response.dto.ts

```diff
@@ -0,0 +1,11 @@
+import type { InboxNotification } from '../utils/types';
+
+export class GetNotificationsResponseDto {
+  data: InboxNotification[];
+  hasMore: boolean;
```

Comment:
@LetItRock I was under the impression we will also add this with a `nextPage` API url. If null, it means there is nothing more to load. [ref](https://docs.stripe.com/api/pagination/search)

Reply:
We can do it ala stripe. We can return the current `url` and a `next_url`.

Reply:
we decided to leave this for now